### ISSUE TITLE
Bug: unable to build with tsc when not using @middy/core

### ISSUE THREAD

**Issue by rreddypally** [aws-powertools/powertools-lambda-typescript#1080]
## Bug description

The code doesn't compile using `tsc` when using the Logger module without middy.

### Expected Behavior

Should be able to use Logger without the need to import middy when its not used. This is possible by importing the submodule via `import { Logger } from "@aws-lambda-powertools/logger/lib/Logger";`. If this is the solution maybe the documentation should reflect that.

### Current Behavior

Middy is required atleast as the dev dependency while compiling with `tsc`. This is not a problem when `esbuild` or `ts-node` is used.

### Steps to Reproduce

Steps:
1. npm install and import `Logger` using `import { Logger } from "@aws-lambda-powertools/logger";`
2. Build project using `tsc`
3. Expect this error `Cannot find module '@middy/core' or its corresponding type declarations.`

`package.json`
```
{
  "scripts": {
    "build": "npm i && tsc logger-demo.ts"
  },
  "dependencies": {
    "@aws-lambda-powertools/logger": "1.2.0",
    "@types/aws-lambda": "8.10.101"
    
  },
  "devDependencies": {
    "typescript": "4.7.4"
  }
}
```
`logger-demo`
```
import { Logger } from "@aws-lambda-powertools/logger/lib/Logger";

const logger = new Logger();
logger.info("Testing logger");
```
`tsconfig.json`
```
{
    "compilerOptions": {
        "experimentalDecorators": true,
        "target": "ES2021",
        "module": "CommonJS"
    }
}
```

### Environment

* **Powertools version used**: 1.2.1
* **Packaging format (Layers, npm)**: npm
* **AWS Lambda function runtime:** node16
* **Debugging logs:**n/a

### Related issues, RFCs

https://github.com/awslabs/aws-lambda-powertools-typescript/issues/1068

**Comment by dreamorosi** [comment]
@obiwabrakenobi, not at this time, however we're happy to review & merge any PR coming from the community to fix the issue in a more timely manner.

**Comment by obiwabrakenobi** [comment]
I'll have a look into it

**Comment by dreamorosi** [comment]
Thank you!

**Comment by ValeryShvyndzikau** [comment]
Hi "AWS Lambda Powertools Team", 
We are going to use Tracer, Logger, and Metrics for observability on our project and we are building our codebase with "tsc". Middy is still under consideration, so it would be nice to avoid redundant dependency on the extra library we don't use.
Could you please share updates related to these 3 issues #1080, #1081, #1068, and when it's planned to be fixed?

**Comment by dreamorosi** [comment]
We just released version `1.5.1` that should fix the issue.

You can read more about the fix in #1225, however now you should be able to transpile your functions with `tsc` without having `@middy/core` in your codebase.

I have tested `v1.5.1.` in the repository [provided here](https://github.com/rreddypally/tracer-demo), and the code now compiles successfully to this:
```ts
"use strict";
Object.defineProperty(exports, "__esModule", { value: true });
const tracer_1 = require("@aws-lambda-powertools/tracer");
// import { captureLambdaHandler } from '@aws-lambda-powertools/tracer'
const tracer = new tracer_1.Tracer();
```

Hope this helps!
